Core/src/org/sleuthkit/autopsy/datamodel/ImageNode.java

```diff
@@ -200,6 +204,27 @@ public String getItemType() {
         return getClass().getName();
     }
 
+    private Boolean checkSchemaVersion() {
+        String sqlStatement = "SELECT a.value creationMajorVersion, b.value creationMinorVersion FROM tsk_db_info_extended a, tsk_db_info_extended b " +
+                              " WHERE a.name = 'CREATION_SCHEMA_MAJOR_VERSION' and b.name = 'CREATION_SCHEMA_MINOR_VERSION';";
+        try (CaseDbQuery query = Case.getCurrentCaseThrows().getSleuthkitCase().executeQuery(sqlStatement);) {
```

Comment:
Perhaps the use of "AS" would make this SQL easier to read?

Reply:
Changed to add AS to select